app/controllers/auth/AuthAction.scala

```diff
@@ -123,7 +132,9 @@ class AuthActionImpl @Inject() (
           for {
             result        <- block(authenticatedRequest)
             updatedResult <- sessionAuditor.auditOnce(authenticatedRequest, result)
-          } yield updatedResult
+          } yield
+            if (singleAccountEnrolmentPresent(enrolments)) updatedResult
```

Comment:
You need a toggle so that the feature can be turned on when mobile app is ready.

Reply:
And also turned off if there are too many issues.

Reply:
Done